util/types/time_test.go

```diff
@@ -406,7 +406,7 @@ func (s *testTimeSuite) TestParseTimeFromNum(c *C) {
 		ExpectDateValue      string
 	}{
 		{20101010111111, false, "2010-10-10 11:11:11", false, "2010-10-10 11:11:11", false, "2010-10-10"},
-		{2010101011111, true, zeroDatetimeStr, true, zeroDatetimeStr, true, zeroDateStr},
+		{2010101011111, false, "0201-01-01 01:11:11", true, zeroDatetimeStr, false, "0201-01-01"},
```

Comment:
I check MySQL's behavior, its result is  "0201-01-01 01:11:11"...
There is a document say datetime range is '1000-01-01 00:00:00' to '9999-12-31 23:59:59', but it seems the implement doesn't obey it.